app/controllers/donors_controller.rb

```
@@ -13,10 +14,9 @@ def get_donations
 		render json: @donor.donations, include: 'claims', status: :ok
 	end
 
-    
 	def create
 		return render json: { error: 'donor email already in use'}, status: :conflict if Donor.exists?({email: donor_params[:email]})
-		@donor = Donor.create(donor_params)
+		@donor = Donor.create!(donor_params)
```

Why this change?  I think this will throw an exception rather than return false now upon failure.  Not sure if that's what we want and I didn't think changes to that method were in the scope of these changes.

---

This was unintentional😬